appserver/deployment/schemas/src/main/resources/glassfish/lib/schemas/web-app_6_0.xsd

```diff
@@ -1,87 +1,82 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2009, 2021 Oracle and/or its affiliates and others.
+    All rights reserved.
+
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License v. 2.0, which is available at
+    http://www.eclipse.org/legal/epl-2.0.
+
+    This Source Code may also be made available under the following Secondary
+    Licenses when the conditions for such availability set forth in the
+    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+    version 2 with the GNU Classpath Exception, which is available at
+    https://www.gnu.org/software/classpath/license.html.
+
+    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+
+-->
+
 <xsd:schema xmlns="http://www.w3.org/2001/XMLSchema"
             targetNamespace="https://jakarta.ee/xml/ns/jakartaee"
             xmlns:jakartaee="https://jakarta.ee/xml/ns/jakartaee"
             xmlns:xsd="http://www.w3.org/2001/XMLSchema"
             elementFormDefault="qualified"
             attributeFormDefault="unqualified"
             version="6.0">
+
   <xsd:annotation>
     <xsd:documentation>
-
-      Copyright (c) 2009, 2021 Oracle and/or its affiliates. All rights reserved.
-      
-      This program and the accompanying materials are made available under the
-      terms of the Eclipse Public License v. 2.0, which is available at
-      http://www.eclipse.org/legal/epl-2.0.
-      
-      This Source Code may also be made available under the following Secondary
-      Licenses when the conditions for such availability set forth in the
-      Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
-      version 2 with the GNU Classpath Exception, which is available at
-      https://www.gnu.org/software/classpath/license.html.
-      
-      SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
-      
+      <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="license.inc" parse="text"/>
```

Comment:
I was under impression that this was updated in
- #23968

already.

To the published (correct) versions instead of sources as is done here.
:confused:

Reply:
It's used here: `com.sun.enterprise.deployment.node.ApplicationNode`.

If it's again the wrong one, could you look at committing the right one?

Reply:
It's used here: `com.sun.enterprise.deployment.node.ApplicationNode`.

If it's again the wrong one, could you look at committing the right one?

Reply:
> could you look at committing the right one?

- #23991

but `web-app_6_0.xsd` seems to not need changes.

Reply:
Could well be, that one can be restored then. Thanks for noticing!

Reply:
I just restored it, thanks again.